src/data/data.js

```diff
@@ -198,6 +190,14 @@ const pageData = {
           },
         ],
       },
+      {
+        component: 'ubication',
+        iframe: {
+          src:
+            'https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2432.0906492940917!2d-75.59378303147989!3d6.255443893561203!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x22c97971d770b4ef!2sAquatic%20Complex%20Atanasio%20Girardot!5e0!3m2!1sen!2sus!4v1599412191860!5m2!1sen!2sus',
+          alt: 'ubicacion de la sede principal',
+        },
+      },
```

Comment:
Why do you move the ubication component?

Reply:
Because it was on top of my component and in the design the map was under.

![image](https://user-images.githubusercontent.com/68561487/93531536-0149fc00-f905-11ea-867f-998a4dcd5c5d.png)